core/server/config/defaults.json

```diff
@@ -57,6 +57,12 @@
             "maxWait": 3600000,
             "lifetime": 3600,
             "freeRetries":99
+        },
+        "content_api_key": {
```

Comment:
These values might need tweaking. Currently you can make 100 incorrect requests in an hour before being blocked, for an hour.

Reply:
Updated to be blocked starting at one hour, and moving to 24

Reply:
That sounds like a good starting point 👍